app/controllers/users_controller.rb

```diff
@@ -0,0 +1,27 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find_by id: params[:id]
```

Comment:
recommend handle incase User do not found
```suggestion
    @user = User.find_by id: params[:id]
    return if @user
    
    flash[:danger] = ... # message not found here
    redirect_to ... # example: redirect_to root_path
```